interface/weno/logsync.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once($GLOBALS["srcdir"] . "/formatting.inc.php");
```

Comment:
remove above line. the formatting.inc.php script is already included automatically by globals.php call

Reply:
actually, this is a bit odd; isn't this script always called from the indexrx.php scipt. If so, why not just include the code from this script in that script?